test/phantom.js

```diff
@@ -0,0 +1,192 @@
+var page = require('webpage').create();
+
+var testsRun = 0;
+var passed = 0;
+var failed = 0;
+
+var tests;
+
+// REPORTER -------------------------------------------------------------------
+
+var reporter = {
+  report: function (test, result) {
+    if (result) {
+      console.log('PASS: ' + test);
```

Comment:
I would add a bit more space/lines/colour to make this stand out. same with the failed state